<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdBlockWidget</class>
 <widget class="QWidget" name="AdBlockWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>704</width>
    <height>482</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Manage Ad Blocker</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="labelRequestsBlocked">
       <property name="text">
        <string>Total number of requests blocked by filters:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelRequestsBlockedValue">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="CheckableTableView" name="tableView"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QToolButton" name="toolButtonAddSubscription">
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>Add Subscription</string>
       </property>
       <property name="popupMode">
        <enum>QToolButton::InstantPopup</enum>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextBesideIcon</enum>
       </property>
       <property name="arrowType">
        <enum>Qt::DownArrow</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonDeleteSubscription">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Delete Subscription</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonCustomFilters">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Custom Filters</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CheckableTableView</class>
   <extends>QTableView</extends>
   <header>Widgets/CheckableTableView.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
